src/holidays/united-states/federal-reserve-bank/add-us-federal-reserve-bank-business-days.ts

```diff
@@ -0,0 +1,59 @@
+// holidays/united-states/federal-reserve-bank/add-us-federal-reserve-bank-business-days.ts
+
+import { isUSFederalReserveBankClosed, PlainDate } from './index';
+
+/**
+ * @name addUSFederalReserveBankBusinessDays
+ * @summary Add or subtract the specified number of US Federal Reserve Bank business days to the given date.
+ *
+ * @description
+ * Add or subtract the specified number of US Federal Reserve Bank business days to the given date.
+ *
+ * @param plainDate - the date to be changed
+ * @param amount - the amount of business days to be added or subtracted. Positive amount will add the number of US
+ * Federal Reserve Bank business days and negative amount will subtract the number of US Federal Reserve Bank business days.
+ * @returns plainDate with the business days added/subtracted
+ */
+export default function (plainDate: PlainDate, amount: number): PlainDate {
+  if (Number.isNaN(amount)) {
+    throw new TypeError('Invalid Amount');
+  }
+
+  const endDate = new Date(plainDate.year, plainDate.month, plainDate.date);
```

Comment:
I feel something like `result` or `resultDate` might be a better name for this.  `endDate` is a little confusing, makes me think its the end of a date range